src/components/App/App.js

```diff
@@ -1,12 +1,34 @@
 import React, {Component} from 'react';
 import './App.css';
-import WebcamCapture from '../WebcamCapture/WebcamCapture.js';
+import WebcamCaptureContainer from '../WebcamCapture/WebcamCaptureContainer.js';
+import ConfirmationBox from '../ConfirmationBox/ConfirmationBox';
 
 class App extends Component {
+	state = {
+		loading: true,
+		likelyItem: null
+	};
+
+	componentDidMount() {
+		this.setState({loading: false});
```

Comment:
Can you actually see the loading state then? I didn't know you could do this - I found this is the docs but probably misunderstood it:
"You may call setState() immediately in componentDidMount(). It will trigger an extra rendering, but it will happen before the browser updates the screen. This guarantees that even though the render() will be called twice in this case, the user won’t see the intermediate state. "

Reply:
Didn't know this - works now